### DISCUSSION TITLE
Does this project support infrastructure expansion?

### DISCUSSION THREAD

**Discussion by lexuanquynh** [meysamhadeli/shop-golang-microservices#88] [Q&A] [answered]
Does this project support infrastructure expansion?
I want to use multiple nodes for Docker. I want to use at least three different containers to be able to support a large number of users.

**Comment by meysamhadeli** [comment, marked as answer]
Currently, this project doesn't have any deployments for scaling. but you can use Kubernetes for scaling this project.
I have the same project in which I do all of these scaling things and you can follow the rules based on this project for scaling on the Kubernetes, but you need a little change because the language is different, but the concepts are the same.
[https://github.com/meysamhadeli/booking-microservices](https://github.com/meysamhadeli/booking-microservices)

**Comment by lexuanquynh** [comment]
thank you very much. You save me a lot of time.

**Reply by meysamhadeli** [reply]
Your welcome :)
